### PR TITLE
add abq metadata introduced in abq version 1.1.2

### PR DESCRIPTION
**Pull request by michaelglass**
snapshots changed because of env vars passed through in new abq version. This just updates the snapshots.

### CODE REVIEW

File: spec/rspec/abq_spec.rb [hunk 1]
```diff
@@ -6,6 +6,13 @@ def stringify_keys(hash)
 end
 
 RSpec.describe RSpec::Abq do
+  before do
```
Comment (michaelglass) [inline spec/rspec/abq_spec.rb:9]: I would have expected this to be enough to remove flakiness but ... apparently it wasn't 😢

File: spec/rspec/abq_spec.rb [hunk 2]
```diff
@@ -88,7 +92,10 @@ def stringify_keys(hash)
       EnvHelper.with_env("ABQ_SOCKET" => "#{host}:#{port}") do
         example.call
       end
-      RSpec::Abq.instance_eval { @socket = nil }
```
Comment (pierrebeaucamp) [inline spec/rspec/abq_spec.rb:91]: Is this an unrelated change?

**Review by doxavore — APPROVED** [review]

File: spec/rspec/abq_spec.rb (hunk 2 above)
Comment (michaelglass) [reply spec/rspec/abq_spec.rb:91]: sorry updated the PR description to reflect the actual changes

overall "fix tests on main"

Comment (doxavore) [reply spec/rspec/abq_spec.rb:91]: This is expanded below by removing all instance variables. That's probably what should have been here before since we use a handful of of them in the Abq singleton class.
